implementations/rust/ockam/ockam_entity/src/worker/profile_sync.rs

```diff
@@ -11,7 +11,7 @@ use tracing::debug;
 
 /// Synchronous worker wrapper around a Profile
 pub struct ProfileSync {
-    ctx: Context,
+    pub(crate) ctx: Context,
```

Comment:
Can we have getter instead?

Reply:
Done. It would be cool if we could somehow generalize / reuse the pattern of 

```rust
block_future(&self.ctx().runtime(), async move {...})
```

To something where we can just pass a closure, like on_runtime(async move {})...
That's tricky for several reasons, I played with it a bit and will have to wait for a later time

Reply:
I think I tried to do that too, and it was tricky